include/mbedtls/cipher.h

```diff
@@ -402,9 +410,9 @@ static inline mbedtls_cipher_mode_t mbedtls_cipher_get_cipher_mode( const mbedtl
  *
  * \param ctx   The context of the cipher. Must be initialized.
  *
- * \return      <ul><li>If no IV has been set: the recommended IV size.
- *              0 for ciphers not using IV or nonce.</li>
- *              <li>If IV has already been set: the actual size.</li></ul>
+ * \return      The recommended IV size if no IV has been set.
+ * \return      \c 0 for ciphers not using IV or nonce.
```

Comment:
Pre-existing: `not using IV or nonce` -> `not using an IV or nonce`.

Reply:
fixed